src/main/resources/data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO users (USER_NAME, GENDER, EMAIL, ROLE, PASSWORD, DATE_OF_BIRTH, ENABLED) VALUES
```

Comment:
Why do you need this file?

Reply:
this is necessary so as not to create a user every time after launching the application

Reply:
Terrible solution. Now it works because you're using a memory H2 database which is dropped when your application stop. If you're using general database like Postgres,MySql, H2 server, and so on, application would throw an exception because it try to create a table which already exists. Read about "ddl-auto" (for example here https://docs.spring.io/spring-boot/docs/1.1.0.M1/reference/html/howto-database-initialization.html)

Reply:
added spring.jpa.hibernate.ddl-auto = create-drop property in application.properties